config/impersonate.yml

```diff
@@ -1,38 +0,0 @@
-defaults: &defaults
```

Comment:
Most of the content in this file is now in `config/secrets` and `config/dpn`; see the `lib/template/...` files for details.